alma/alma-8.x/alma-8.7-hpc/install.sh

```diff
@@ -32,10 +35,10 @@ $ALMA_COMMON_DIR/install_lustre_client.sh "8"
 ./install_intel_libs.sh
 
 # cleanup downloaded tarballs - clear some space
-rm -rf *.tgz *.bz2 *.tbz *.tar.gz *.run *.deb *_offline.sh
-rm -rf /tmp/MLNX_OFED_LINUX* /tmp/*conf*
-rm -rf /var/intel/ /var/cache/*
-rm -Rf -- */
+#rm -rf *.tgz *.bz2 *.tbz *.tar.gz *.run *.deb *_offline.sh
```

Comment:
Once the current test build runs and is verified, these lines to cleanup install will be uncommented